app/src/main/java/com/github/se/project/ui/message/Chat.kt

```diff
@@ -0,0 +1,36 @@
+package com.github.se.project.ui.message
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.platform.LocalContext
+import com.github.se.project.model.chat.ChatViewModel
+import com.github.se.project.ui.navigation.NavigationActions
+import com.github.se.project.ui.navigation.Screen
+import io.getstream.chat.android.compose.ui.messages.MessagesScreen
+import io.getstream.chat.android.compose.ui.theme.ChatTheme
+import io.getstream.chat.android.compose.viewmodel.messages.MessagesViewModelFactory
+
+@Composable
+fun ChatScreen(
+    navigationActions: NavigationActions,
+    chatViewModel: ChatViewModel,
+) {
+  val channelID = chatViewModel.currentChannelID.collectAsState()
+
+  if (channelID.value == null) {
```

Comment:
This ensures the user is not allowed into the chat screen without a valid channel, LGTM.